src/platform/plugins/shared/data/server/search/strategies/esql_async_search/esql_async_search_strategy.ts

```diff
@@ -197,7 +197,12 @@ export const esqlAsyncSearchStrategyProvider = (
      */
 
     cancel: async (id, options, deps) => {
-      logger.debug(`cancel ${id}`);
+      if (options.isStored) {
+        logger.debug(`[esqlAsyncSearch] can't be canceled ${id} ${JSON.stringify(options)}`);
```

Comment:
Unfortunately it does look like this prevents the searches from being canceled when you delete a background search